library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/BaseWireframeMapperTest.kt

```diff
@@ -96,6 +97,10 @@ internal abstract class BaseWireframeMapperTest {
         return textView.text.toString()
     }
 
+    protected fun Forge.aClass(): Class<*> {
+        return anElementFrom()
```

Comment:
it will crash if not overriden, right? maybe we just need to throw explicit exception here? because if it throws by calling this, it will be quite non-evident why it is like that.

Reply:
this is left over 🤦